extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.tracking.js

```diff
@@ -66,66 +49,46 @@ define('wikia.articleVideo.featuredVideo.tracking', ['wikia.tracker'], function
 
 			var relatedPlugin = playerInstance.getPlugin('related');
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the declaration of the unused 'relatedPlugin' variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedVariable)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "relatedPlugin" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)